griptape/artifacts/video_artifact.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from attrs import define, field
+
+from griptape.artifacts import BlobArtifact
+
+
+@define
+class VideoArtifact(BlobArtifact):
+    """Stores video binary data and relevant metadata.
+
+    Attributes:
+        value: The video binary data.
+        mime_type: The video MIME type.
+        resolution: The resolution of the video (e.g., 1920x1080).
+        duration: Duration of the video in seconds.
```

Comment:
which of these do you feel confident you can have in the initial version (for example, it sounded like a target duration may not be something easy to get)? Are there other params that the providers expose that we should be taking into consideration? For example, I would anticipate framerate to be a popular parameter so that we can have everyone's favorite NTSC 59.94!

Reply:
I'm going to somehow include most of the things  [Jason](https://griptape-ai.slack.com/archives/C05T3HLBH52/p1727295829173599?thread_ts=1727292898.313379&cid=C05T3HLBH52) mentioned, I don't believe that there is a way to bring audio with these videos, so probably no subtitles for initial version.